client.go

```diff
@@ -125,6 +125,7 @@ type Client struct {
 	udBeforeRequest    []RequestMiddleware
 	preReqHook         PreRequestHook
 	afterResponse      []ResponseMiddleware
+	udAfterResponse    []ResponseMiddleware
```

Comment:
@lggomez `udAfterResponse` initialize it when a resty client gets created.
Like -
https://github.com/go-resty/resty/blob/d2ba00f5a573e785d8d90e05433d3f4ef4ad3665/client.go#L969